src/schemas.ts

```diff
@@ -2,77 +2,77 @@
 // The schema-validator will pre-load these
 // This file will need to be manually updated with all schemas that are ready for consumption
 //
-import { JSONSchema7 } from 'json-schema';
```

Comment:
Please don't change the quote style like this in just a few files file. JSON is in " and the TypeScript code uses single quotes for string.

Reply:
Hmm, pretty weird bug. My default prettier/ vs code config is set to single quotes but it just loves to use doubles for some reason. Anyway resolved the single quotes format; going to try reinstalling the extentions I have to solve this.